lightbox.js

```diff
@@ -164,7 +164,7 @@ function init(options) {
     images[i] = new LightboxImage($img.data('src'), i);
   });
 
-  $context.find('.js-lightbox').on('click', function(e) {
+  $context.find(':not(a) > .js-lightbox').on('click', function(e) {
```

Comment:
i agree, i wanted to do
```js
$context.find(".js-lightbox").filter(function(idx, e) {
  return $(e).parents("a").length === 0;
});
```
BUT got this
`VM36927:1 Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '[object HTMLPictureElement]' is not a valid selector.`